### PR TITLE
feat: link component

### PR DESCRIPTION
**Pull request by jessywork**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/components/link/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+**npm**
+
+```sh
+npm i @douro-ui/link react@18.2.0 react-dom@18.2.0
+```
+
+## Usage
+
+```js
+import { Link } from '@douro-ui/link';
```
Comment (joaoprcosta) [inline packages/components/link/README.md:27]: When you export `default as something`, you import without brackets.

```suggestion
import Link from '@douro-ui/link';
```

File: packages/components/link/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import type { LinkProps } from '@douro-ui/link';
+```
+
+or
+
+```js
+import { Link, LinkProps } from '@douro-ui/link';
```
Comment (joaoprcosta) [inline packages/components/link/README.md:34]:
```suggestion
import Link { LinkProps } from '@douro-ui/link';
```

File: packages/components/link/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "url": "https://github.com/Douro-ui/design-system"
+  },
+  "author": "Metyis Porto",
+  "bugs": "https://github.com/Douro-ui/design-system/issues",
+  "license": "MIT",
+  "scripts": {
+    "build": "vite build"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.11.4",
```
Comment (joaoprcosta) [inline packages/components/link/package.json:18]: @JuniorDiasOliveira should it be peer dependency?

File: packages/components/link/src/Link.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import type { LinkProps } from './link.types';
+import { LinkStyled } from './link.styles';
+import { useTheme } from '@douro-ui/react';
+import { FC } from 'react';
+
+const Link: FC<LinkProps> = ({ underline, fontSize, ...props }) => {
+  const theme = useTheme();
+
+  return (
+    <LinkStyled
```
Comment (joaoprcosta) [inline packages/components/link/src/Link.tsx:10]: A link typically has a href, target, and possibility accessibility attributes 'Aria'. 
Please read a little bit of this article: https://amberwilson.co.uk/blog/are-your-anchor-links-accessible/

File: packages/components/link/src/__tests__/Link.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import { render, screen } from '@testing-library/react';
+import Link from '../Link';
+import { ThemeProvider } from '@douro-ui/react';
+
+describe('<Link />', () => {
+  it.only('renders correctly with text', () => {
```
Comment (joaoprcosta) [inline packages/components/link/src/__tests__/Link.spec.tsx:6]:
```suggestion
  it('renders correctly with text', () => {
```

File: packages/components/link/src/Link.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+      underline={underline}
+      fontSize={fontSize}
```
Comment (joaoprcosta) [inline packages/components/link/src/Link.tsx:12]: Should import via theme.

File: packages/components/link/src/Link.tsx (hunk 4 above)
Comment (joaoprcosta) [reply packages/components/link/src/Link.tsx:9]: Ignore this comments. Passing `...props` is enough.

File: packages/components/link/src/link.styles.ts [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  width: 0.063rem;
+  height: 0.063rem;
+  padding: 0;
+  margin: -0.063rem;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  border: 0;
+`;
+
+// const linkDisabledStyles = ({ styledSelectedColor }: TabsStyledProps) => css`
```
Comment (joaoprcosta) [inline packages/components/link/src/link.styles.ts:15]: Missing commented code.

File: packages/components/link/package.json (hunk 3 above)
Comment (JuniorDiasOliveira) [reply packages/components/link/package.json]: yes, Jessica will change it, thanks!

**Review by JuniorDiasOliveira — APPROVED** [review]

**Review by joaoprcosta — APPROVED** [review]

**Pull request merged**